core/cmd/cli/conf/xendorser_cli.yaml

```diff
@@ -0,0 +1,17 @@
+# endorseService Info
```

Comment:
How can I config TestNet or OpenNet？

Reply:
Use two different configuration files. When using, specify the configuration file path